### DISCUSSION TITLE
Collection Permission API

### DISCUSSION THREAD

**Discussion by cloud303-cholden** [outline/outline#5443] [Ideas]
Is there a way to change a collection's permission using the API? I am wanting to create private collections with `read_write` permission. I'm only seeing `permission` for filtering by permission in requests and in collection responses.

**Comment by cloud303-cholden** [comment]
Perhaps this is related to this: [link](https://github.com/outline/outline/discussions/4382)?

**Reply by cloud303-cholden** [reply]
After further investigation, it appears that my use case would be possible if `/api/collections.add_user` supported passing in the `permission` for the added user to have.

**Reply by tommoor** [reply]
https://github.com/outline/outline/blob/fbd16d4b9a4fefe349c0e6ea850ecdd8a95839f4/server/routes/api/collections.ts#L631

Although it's not documented `permission` is supported in `collections.update` endpoint.

**Reply by cloud303-cholden** [reply]
Thanks for the reply. It looks like `collections.add_user` also supports this parameter, although it is not documented.
